### ISSUE TITLE
Sync workflow issues

### ISSUE THREAD

**Issue by DecaTec** [vauvenal5/yaga#150]
I have set up the Nextcloud app to:
* Auto upload images to a "photos" folder in my Nextcloud.
* Keep the photos after upload where they are (`DCIM/Camera` folder)

Now, I wanted to implement the sync workflow as described here: https://vauvenal5.github.io/yaga-docs/sync_flow/

> In case you configured your auto-upload to keep images where they are, then you can specifiy for the Remote Path the same server directory as for the auto-upload and for the Local Path you can set the local folder where you keep your images. This way Nextcloud Yaga will know not to re-download those images. Other images you download will be saved to Nextcloud Yaga's app folder.

Yaga configuration:
* Global settings > Root mapping:
    * Remote path: The auto upload folder in my Nextcloud
    * Local path: `DCIM/camera` (where the photos are saved as default)
* Settings home view:
    * Path: The auto upload folder in my Nextcloud

The images are shown in the Yaga app as expected, but there seems to be an issue with downloading/deleting photos in the Yaga app:
* If I want to delete in image locally, the image gets not deleted.
* If I want to download an image which is not present locally (by opening the image in full screen), this also doesn't work: The image is shown in a reduced resolution and the progress bar runs endlessly.

Is there a fault in my configuration?

**Comment by DecaTec** [comment]
I tried a few things and it seems to be an issue with the Android permission system: When enabling full access for the app in Android's settings, it works as expected.

Maybe the app needs additional permissions for the camera?

Pixel 5/Android 11.

**Comment by vauvenal5** [comment]
Yes the app is still targeting Android 10 because of the most recent permission changes. There will be quite some changes in the way it handles files which are outside of its own app folder.

**Comment by vauvenal5** [comment]
See also #144
